### PR TITLE
Po image input

### PR DESCRIPTION
**Pull request by shruthipatil11**
image and input components

### CODE REVIEW

File: src/app/components/image/image.css [hunk 1]
```diff
@@ -0,0 +1,16 @@
+.image--round {
+  border-radius: 50%;
+  width: 15rem;
```
Comment (manojsarna) [inline src/app/components/image/image.css:3]: Look at this , please use correct naming for the classes ( esp. if you are using BEM Model )

File: src/app/components/input/input.css [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  left: 14px;
+}
+
+.input--success > input {
+  border: 2px solid var(--INPUT-SUCCESS);
+}
+.input--success .input__label {
+  color: var(--INPUT-SUCCESS);
+}
+
```
Comment (manojsarna) [inline src/app/components/input/input.css]: looks good!!

File: src/app/docs/docs.html [hunk 3]
```diff
@@ -252,6 +252,168 @@ <h3>Badge on Icons</h3>
       </section>
 
       <!-- Badge ends -->
+
+      <!-- images start-->
```
Comment (manojsarna) [inline src/app/docs/docs.html:232]: check for spacing!!

File: src/app/docs/docs.html [hunk 4]
```diff
@@ -252,6 +252,168 @@ <h3>Badge on Icons</h3>
...
+            <img
+              class="image--responsive"
+              src="../../assets/images/image-responsive.jpg"
+              alt="Responsive image"
+            />
+          </figure>
+        </div>
+
+        <span>
+          Class Modifier:
```
Comment (manojsarna) [inline src/app/docs/docs.html:252]: here too

**Review by manojsarna — CHANGES_REQUESTED** [review]
Overall Lokks Good!!

Have Commented , Do Check It!!

File: src/app/components/image/image.css (hunk 1 above)
Comment (shruthipatil11) [reply src/app/components/image/image.css:3]: This is right Manoj . image--round means modifying the image.

**Review by Mahi24-prog — APPROVED** [review]
Make it fully responsive. 
Other than that it's looking good

File: src/app/components/input/input.css [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+  min-width: 5rem;
+  outline: none;
+}
+
+.input-field .input__label {
+  position: absolute;
+  left: 0;
+  bottom: 33px;
+  background-color: white;
+  left: 14px;
```
Comment (anuj-kumary) [inline src/app/components/input/input.css]: Is there any specific reason to use PX?

**Review by manojsarna — APPROVED** [review]
Looks Great!!

File: src/app/components/input/input.css (hunk 5 above)
Comment (shruthipatil11) [reply src/app/components/input/input.css]: changed to px.

File: src/app/docs/docs.html [hunk 6]
```diff
@@ -252,6 +252,168 @@ <h3>Badge on Icons</h3>
...
+      <div class="components" id="image">
+        <h2 class="component__heading">Responsive images</h2>
+        <p>
+          Images are made responsive with .image--responsive . This applies
+          max-width: 100% and height: auto to the image so that it scales with
+          the parent width.
+        </p>
+        <h3>Responsive Image</h3>
+        <div class="components__parts">
+          <figure class="image__container">
```
Comment (PuneetP16) [inline src/app/docs/docs.html]: Use consistent BEM naming convention, for blocks, name it as block_name or block-name.
For elements, name it as block-name__element-name or block_name__element_name.

Element should be named as block if it is independent and can be reused in other components or projects.

## for more information, go to 
http://getbem.com/introduction/

## for reference
https://en.bem.info/methodology/naming-convention/

**Review by PuneetP16 — APPROVED** [review]
Looks good to merge. left few suggestions

File: src/app/docs/docs.html (hunk 3 above)
Comment (shruthipatil11) [reply src/app/docs/docs.html:232]: done

File: src/app/docs/docs.html (hunk 4 above)
Comment (shruthipatil11) [reply src/app/docs/docs.html:252]: done

File: src/app/docs/docs.html (hunk 6 above)
Comment (shruthipatil11) [reply src/app/docs/docs.html]: Updated the code, thanks for info.

**Pull request merged**